### ISSUE TITLE
[Feature request] Docker image

### ISSUE THREAD

**Issue by nielsfaber** [zwave-js/zwave-js-server#90]
I'd love to see a docker image of this project, to provide a better "out of the box" experience for some users.
I don't run hass.io or supervisor, so I'm currently stuck with ZwaveJS2Mqtt since I can run it as a docker container.

Some example Dockerfile (which I have used successfully to dockerize one of my own node.js apps):
```
FROM node:12

RUN mkdir -p /home/node/app/node_modules && chown -R node:node /home/node/app

WORKDIR /home/node/app

COPY package*.json ./

USER node

RUN npm install

COPY --chown=node:node . .

CMD [ "node", "app.js" ]
```
It probably needs some grooming here and there, but overall it should be fairly simple.

**Comment by zeddski** [comment]
There is a, I think unrelated, docker image available by "kpine":
https://hub.docker.com/r/kpine/zwave-js-server

I wasn't able to get it running, maybe it works for you.

**Comment by frenck** [comment]
I'm a bit confused, the ZWaveJS2MQTT Docker containers are available, right?
Despite the suggested name; the MQTT part is optional.

**Comment by nielsfaber** [comment]
> I'm a bit confused, the ZWaveJS2MQTT Docker containers are available, right?
> Despite the suggested name; the MQTT part is optional.

I was trying to follow [this](https://community.home-assistant.io/t/switching-from-openzwave-beta-to-zwavejs2mqtt/276724) guide.
So far it is the best available documentation I could find on how to set up ZwaveJS in HA using ZWaveJS2MQTT (which seems like the way to go until we can run the zwave-js-server from docker).

As you said, the name suggests (implies?) that the application communicates through MQTT.
I installed the docker image referred in the guide, the web interface doesn't seem to offer any alternatives to MQTT.
The guide doesn't cover any setup of the ZWaveJS2MQTT part, maybe that's the missing link here.
I guess we are struggling due to the lack of documentation on how to set things.

Anyway, the zwave-js-server seems very lean compared to ZWaveJS2MQTT.
Some users (including myself) may prefer it.
I'd say my request remains valid despite an alternative being available.

**Comment by frenck** [comment]
![image](https://user-images.githubusercontent.com/195327/107117296-c461b000-6879-11eb-92f3-0ef3edd15a91.png)

Screenshot from my ZWaveJS2MQTT instance:

The WS Server is enabled (= ZWave JS Server). The MQTT gateway is disabled.

**Comment by nielsfaber** [comment]
> Screenshot from my ZWaveJS2MQTT instance:

Oh wow!
For me it showed a field "Polling interval" instead of the "WS Server".
Turns out I was running an image with the wrong build tag.
I came across [this](https://community.home-assistant.io/t/lets-start-talking-about-the-new-z-wave-js-integration/269731/38?u=neliss) post which clarifies it.
Thanks for the trigger @frenck !

We can close this request if you suggest people should stick with ZWaveJS2MQTT for docker environments.
It seems to be doing the job as well (finally) 👍
